djcelery_email/tasks.py

```diff
@@ -44,7 +44,10 @@ def send_emails(messages, backend_kwargs=None, **kwargs):
     messages = [email_to_dict(m) for m in messages]
 
     conn = get_connection(backend=settings.CELERY_EMAIL_BACKEND, **combined_kwargs)
-    conn.open()
+    try:
+        conn.open()
+    except Exception as exp:
+        logger.info(str(exp))
```

Comment:
I think `logger.exception` might be a better choice here, as it allows you to pass in a custom message and still maintain the exception information.

[python exception logging](http://stackoverflow.com/q/5191830/359284)